build/Uno.Foundation.Logging.nuspec

```diff
@@ -16,17 +16,7 @@
 
   <files>
     <file src="..\src\Common\uno.png" target="/" />
-
     <file src="..\src\Uno.Foundation.Logging\bin\Uno.Foundation.Logging\Release\netstandard2.0\*.*" target="lib\netstandard2.0" />
```

Comment:
Uno.Foundation.Logging doesn't appear to have any platform-specific code and shouldn't need to target anything other than netstandard2.0, but let's see what CI says :)